.firebase/hosting.YnVpbGQ.cache

```diff
@@ -0,0 +1,15 @@
+asset-manifest.json,1588697398498,11d0d1b1dffc8e20ea0fad3b5d2519fd84a8955e62acae88a32b3500bbcecd38
```

Comment:
You'll need to add .firebase/ to .gitignore

Reply:
Done